sections/futures/Trade/Trade.tsx

```diff
@@ -255,10 +255,10 @@ const Trade: React.FC = () => {
 				txHash: orderTxn.hash,
 				onTxConfirmed: () => {
 					onLeverageChange('');
-					setTimeout(() => {
-						futuresMarketPositionQuery.refetch();
+					setTimeout(async () => {
 						futuresPositionHistoryQuery.refetch();
 						marketQuery.refetch();
+						refetch();
```

Comment:
Would it make sense to refetch "contract" things immediately, and to refetch the "subgraph" things after some delay?